stable/rethinkdb/templates/rethinkdb-cluster-service.yaml

```diff
@@ -0,0 +1,22 @@
+apiVersion: v1
+kind: Service
+metadata:
+  name: "{{ template "fullname" . }}-cluster"
+  labels:
+    app: "{{ template "name" . }}-cluster"
+    chart: "{{ .Chart.Name }}-{{ .Chart.Version }}"
+    heritage: {{ .Release.Service | quote }}
+    release: {{ .Release.Name | quote }}
+  annotations:
+    service.alpha.kubernetes.io/tolerate-unready-endpoints: "true"
```

Comment:
Ah okay, thanks @unguiculus - I assume the "alpha" is just left behind and so this isn't actually an alpha feature.